utils/streams/index.mjs

```diff
@@ -1,73 +1,103 @@
+  } else {
+    throw new Error('Function outputFile not found path key');
+  }
 });
 
-actions.add('convertFromFile', 'Convert csv data to json', (filePath) => {
-  csvToJsonTransfer(fs.createReadStream(filePath), process.stdout);
+actions.add('convertFromFile', 'Convert csv data to json. Need file path (--file)', ({ file }) => {
+  if (file) {
+    csvToJsonTransfer(fs.createReadStream(file), process.stdout, errorHandler);
+  } else {
+    throw new Error('Function convertFromFile not found file key');
+  }
 });
 
-actions.add('convertToFile', 'Convert csv data to json and save into file', filePath => {
-  csvToJsonTransfer(fs.createReadStream(filePath), fs.createWriteStream(filePath.replace('.csv', '.json')));
+actions.add('convertToFile', 'Convert csv data to json and save into file. Need file path (--file)', ({ file }) => {
+  if (file) {
+    csvToJsonTransfer(fs.createReadStream(file), fs.createWriteStream(file.replace('.csv', '.json')), errorHandler);
+  } else {
+    throw new Error('Function convertToFile not found file key');
+  }
 });
 
-actions.add('bundleCSS', 'Bundle CSS', path => {
-  const outStr = fs.createWriteStream(path + '/bundle.css');
-  fs.readdirSync(path).forEach(item => {
-    const inStr = fs.createReadStream(`${ path }/${ item }`);
-    simpleTransfer(inStr, outStr);
-  });
-  simpleTransfer(
-    request(EPAM_CSS_FILE),
-    fs.createWriteStream(path + '/bundle.css', { flags: 'a' })
-  );
+actions.add('bundleCSS', 'Bundle CSS. Need path (--path)', ({ path }, errorHandler) => {
+  if (path) {
+    const outStr = fs.createWriteStream(path + '/bundle.css');
+    const inStrs = [];
+    fs.readdirSync(path).forEach(item => {
+      if (/.css$/.test(item)) {
+        inStrs.push(fs.createReadStream(`${ path }/${ item }`));
+      }
+    });
+    inStrs.push(request(EPAM_CSS_FILE));
+    combinedTransfer(inStrs, outStr);
+  } else {
+    throw new Error('Function bundleCSS not found path key');
+  }
 });
 
-const params = minimist(process.argv.slice(2));
+const params = minimist(process.argv.slice(2), OPTIONS);
 const paramsKeys = Object.keys(params);
-const firstKey = paramsKeys[PARAMS_INDEXES.FIRST];
-const secondKey = paramsKeys[PARAMS_INDEXES.SECOND];
+const firstParam = paramsKeys[PARAMS_INDEXES.FIRST];
+const secondParam = paramsKeys[PARAMS_INDEXES.SECOND];
 
-switch (firstKey) {
-  case COMMANDS.ACTION:
-  case SHORT_COMMANDS.ACTION:
-    switch (secondKey) {
-      case COMMANDS.HELP:
-      case SHORT_COMMANDS.HELP:
-        console.log(actions.help(params[firstKey]));
-        break;
-      default:
-        try {
-          actions.make(params[firstKey], [params[secondKey]]);
-        } catch (exception) {
-          console.log(exception);
-          console.log(helpShow(COMMANDS.ACTION));
-          console.log(actions.help(params[firstKey]));
-        }
+try {
+  if (firstParam && paramsKeys[PARAMS_INDEXES.FIRST] === COMMANDS.ACTION) {
+    const action = params[firstParam];
```

Comment:
It might be like
`const action = params[COMMANDS.ACTION]`
So afterall it won't be necessary to define _paramsKeys_, _firstParam_, _secondParam_